mipidsi/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- rename all `Builder::with_*` methods to versions without the `with_` prefix
```

Comment:
Perhaps we should add the reason for this change to the changelog and migration guide, which would be to conform with the Rust API guidelines. Otherwise the users might think that we just like to break their code :wink:

Reply:
done